.circleci/config.yml

```diff
@@ -687,6 +693,7 @@ jobs:
             ./script/create_release.sh ${CIRCLE_TAG} << pipeline.parameters.KUBEAPPS_REPO >>
   local_e2e_tests:
     machine: true
+    resource_class: large
```

Comment:
Added the URL to CircleCI docs.
Regarding #4170, if you think that it will help with "spending" better the minutes in CircleCI, I would say go for it. Probably we will need them 😄